From 8863630367899806e5699b8d3ea61b7695585364 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 08 Nov 2021 18:29:52 +0200
Subject: [PATCH] [SW-0] gaudi2: fix compilation

Currently macros used in common/common.c file is defined in arm/common.h
which fails compilation. Move them to common/common.h.

Change-Id: I1c81d6793472fc2f0d48b4837cc63d9d8fad3722
---

diff --git a/arch/gaudi2/arm/common.h b/arch/gaudi2/arm/common.h
index 69108ca..2e0ac15 100644
--- a/arch/gaudi2/arm/common.h
+++ b/arch/gaudi2/arm/common.h
@@ -109,9 +109,6 @@
 #define PCIE_PHY_PROTOCOL0_EXT_MPLLB_BANDWIDTH_LOW	0x1C3
 #define PCIE_PHY_PROTOCOL0_EXT_MPLLB_BANDWIDTH_HIGH	0x1C3
 
-#define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_MODE_G3_CONF	0x5555
-#define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_SEL_G4_CONF	0xFF
-
 #define I2C_SLAVE_DEFAULT_INTR_MASK			0x48FF
 
 #define FW_WORD_NUM					0x2A00 /* FW_SIZE / 2 */
diff --git a/arch/gaudi2/common/common.h b/arch/gaudi2/common/common.h
index 156b9e7..4c2adfb 100644
--- a/arch/gaudi2/common/common.h
+++ b/arch/gaudi2/common/common.h
@@ -36,6 +36,9 @@
 	#define HMMU_HIF_ENABLED_MASK 0xFFFF
 #endif
 
+#define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_MODE_G3_CONF	0x5555
+#define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_SEL_G4_CONF		0xFF
+
 #define ARC0_ICCM_FULL_BASE			0x1000007FF7000000ull
 #define ARC0_ICCM_SIZE				0x20000			/* 128KB */
 #define ARC0_DCCM_FULL_BASE			0x1000007FF7020000ull
